src/dashboard/src/components/header/Header.tsx

```diff
@@ -37,7 +37,7 @@ export const Header: React.FC = () => {
 
   return (
     <>
-      <header className={`${style.header} ${isClient && isDashboardView && style.filterPadding}`}>
+      <header className={`${style.header} ${isClient || isHSB && isDashboardView && style.filterPadding}`}>
```

Comment:
Is there any value in `isClient || isHSB`?  Should it have `style.filterPadding` if it's the dashboard view?

Reply:
You're right, those checks are not necessary.  The filterPadding class should always be there on the dashboard view.  Removed them.